spec/amp-cors-requests.md

```diff
@@ -145,12 +145,7 @@ For example, endpoints should allow requests from:
 - Google AMP Cache subdomain: `https://<publisher's domain>.cdn.ampproject.org` <br>(for example, `https://nytimes-com.cdn.ampproject.org`)
 - Cloudflare AMP Cache: `https://<publisher's domain>.amp.cloudflare.com`
 
-{% call callout('Read on', type='read') %}
-For information on AMP Cache URL formats, see these resources:
-
-- [Google AMP Cache Overview](https://developers.google.com/amp/cache/overview)
-- [Cloudflare AMP Cache](https://amp.cloudflare.com/)
-  {% endcall %}
+Read on: For information on AMP Cache URL formats, see these resources: [Google AMP Cache Overview](https://developers.google.com/amp/cache/overview) and [Cloudflare AMP Cache](https://amp.cloudflare.com/).
```

Comment:
I think the words `Read on: ` can be dropped here.

Reply:
Nope. That's an amp.dev shortcut which will result in a visual callout. The good thing about this is that it's still readable in markdown.